automation_tools/__init__.py

```diff
@@ -1741,9 +1741,11 @@ def apply_hotfix():
         run('/root/hotfix.sh')
 
 
-def upstream_install(admin_password=None, run_katello_installer=True,
-                     puppet4=True):
+def upstream_install(admin_password=None, run_katello_installer=True):
```

Comment:
we have to have same signature for all ```*_install()``` puppet4 param couldn't be provided anyway, so removing